### DISCUSSION TITLE
Support for LaTex?

### DISCUSSION THREAD

**Discussion by mafsi** [dracula/dracula-theme#756] [Ideas] [answered]
Hi there!
Any chance to have support for LaTeX?

**Comment by benknoble** [comment]
How? Are you looking for a theme for a LaTeX editor or for LaTeX-produced documents (_e.g._, Beamer presentations)?

**Comment by mafsi** [comment]
Beamer presentation… Never thought about this! Very interesting idea! I was referring about VS Code support for LaTeX. The display of the code can be improved, especially in \autocite[post]{key}. But please keep the idea about the Beamer presentation also. LaTeX beamer is missing good themes.

**Reply by benknoble** [reply, marked as answer]
If you have issues with VS Code, I'd suggest filing an issue at https://github.com/dracula/visual-studio-code.
